opencensus/stats/measure_to_view_map.py

```diff
@@ -93,7 +93,7 @@ def register_view(self, view, timestamp):
         self._measure_to_view_data_list_map[view.measure.name].append(
             ViewData(view=view, start_time=timestamp, end_time=timestamp))
 
-    def record(self, tags, measurement_map, timestamp):
+    def record(self, tags, measurement_map, timestamp, attachments=dict()):
```

Comment:
Default value for `attachments` should be `None`.

Reply:
Done